Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/CertificateManagerImpl.java

```diff
@@ -339,31 +343,51 @@ public PrivateKey getDefaultPrivateKey(String alias) throws CertificateManagerEx
         return null;
     }
 
+    private Map<String, PrivateKeyEntry> getPrivateKeyEntries() throws CertificateManagerException {
+        Map<String, PrivateKeyEntry> pkEntries = new HashMap<>();
+        final String password = getKeyStoreSystemProperties().get(KEY_STORE_PASSWORD_KEY);
+        if (password != null) {
+            try {
+                Enumeration enumeration = keyStore.aliases();
+                while(enumeration.hasMoreElements()) {
+                    String alias = (String)enumeration.nextElement();
+                    PrivateKeyEntry pkEntry = (PrivateKeyEntry) keyStore.getEntry(alias,
+                        new KeyStore.PasswordProtection(password.toCharArray()));
+                    pkEntries.put(alias, pkEntry);
+                }
+
+            } catch (final NoSuchAlgorithmException | KeyStoreException | UnrecoverableEntryException ex) {
+                LOG.error("Error initializing Private Keys: {}", ex.getLocalizedMessage(), ex);
```

Comment:
Optional: No need to log since we throw the exception here. The caller will catch and log it.